pkg/client/client.go

```diff
@@ -54,6 +54,8 @@ type V1Client struct {
 	CreateClusterGroupFn func(*models.V1ClusterGroupEntity) (string, error)
 	GetClusterGroupFn    func(string) (*models.V1ClusterGroup, error)
 	GetApplicationFn     func(string) (*models.V1AppDeployment, error)
+	UpdateClusterGroupFn func(string, *models.V1ClusterGroupHostClusterEntity) error
```

Comment:
Could you please group these functional objects together by Object type i.e. application is separated from cluster group for example.

Reply:
updated